ConsoleBank/ConsoleBank/Currencies/CurrencyInfo/CurrencyInfo.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Currencies
+{
+    public class CurrencyInfo : ICurrencyInfo
```

Comment:
такие классы лучше обзывать сервисами, имя вполне норм, только постфикса не хватает
```suggestion
    public class CurrencyInfoService : ICurrencyInfoService
```
CurrencyInfo это звучит как объект, поэтому лучше явно указать что это сервис с логикой